src/test/scala-2.11/org/apache/james/gatling/smtp/scenari/NoAuthenticationNoEncryptionBigBodyScenario.scala

```diff
@@ -0,0 +1,31 @@
+package org.apache.james.gatling.smtp.scenari
+
+import io.gatling.core.Predef._
+import org.apache.james.gatling.control._
+import org.apache.james.gatling.jmap.scenari.common.Configuration._
+import org.apache.james.gatling.smtp.PreDef._
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+class NoAuthenticationNoEncryptionBigBodyScenario extends Simulation {
+
+  val myRandom = Random.alphanumeric
+
+  def generateMessage() : String =
+    myRandom take (1024 * 1024) mkString
```

Comment:
so this is one big line of 1M? I would be curious to see if it changes something if you put instead many small lines

Reply:
I'm not sure that my intent is clear :)